R/onAttach.R

```diff
@@ -20,6 +20,8 @@
   if (!isTRUE(getOption("datatable.quiet"))) {   # new option in v1.12.4, #3489
     packageStartupMessage("data.table ", v, if(dev)paste0(" IN DEVELOPMENT built ",d,g),
                           " using ", getDTthreads(verbose=FALSE), " threads (see ?getDTthreads).  Latest news: r-datatable.com")
+    if (gettext("TRANSLATION CHECK", domain='R-data.table') != "TRANSLATION CHECK")
+      packageStartupMessage(gettext("**********\n", "Running data.table in English; package support is available in English only. When searching for online help, be sure to also check for the English error message. This can be obtained by looking at the po/R-<locale>.po and po/<locale>.po files in the package source, where the native language and English error messages can be found side-by-side"))
```

Comment:
Should that English be non-English?
If so, how about more detail:
```data.table is running in non-English mode; i.e. errors and message will be displayed in Chinese.  When reporting problems or searching for help though, please use the English translation which can be found in data.table's source `po` directory:  po/R-<locale>.po and po/<locale>.po. You can also start data.table/R in English mode by doing ...```

Reply:
if this error is triggered, it will be translated; in Chinese it'll say translate. In future, it could say other languages.